ocs_ci/framework/main.py

```diff
@@ -94,10 +95,16 @@ def main(argv=None):
     init_ocsci_conf(arguments)
     pytest_logs_dir = utils.ocsci_log_path()
     utils.create_directory_path(framework.config.RUN['log_dir'])
+    launch_name = f"Tier run from {getuser()} RunId: {framework.config.RUN['run_id']}"
```

Comment:
Yes I understand but that is available only for jenkins and we need something here in ocs-ci so it has to be written fresh if i am not mistaken. If its easier for you to add a function by copy/pasting the code in ocs-ci please do so, then we will have it in only one place. The tags and other things can be added as needed with follow on PRs.

Reply:
We could try to pull that value (which would work in the event of a jenkins execution) and if it doesn't exist we could construct the name. We do something like this for email subject/polarion testrun name: https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/utils.py#L1181-L1230

This won't work entirely as is because it includes information we would rather tag launches with than include in the launch name, but this at least gives you a good idea of what we do to retrieve this data.

Reply:
+1 I am picking up the above function and putting in RUN config.

Reply:
Doesn't it make more sense to stay in the `REPORTING` section?

Reply:
I didn't have to use anything.